src/test/kotlin/interactor/GetHighestDifferentInApartmentRentTest.kt

```diff
@@ -0,0 +1,39 @@
+package interactor
+
+import dataSource.FakeDataSource
+import model.CityEntity
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+class GetHighestDifferentInApartmentRentTest {
+    private lateinit var highestDifferentInApartmentRent: GetHighestDifferentInApartmentRent
+    private lateinit var fakeDataSource: FakeDataSource
+
+    @BeforeEach
+    fun setup(){
+        fakeDataSource = FakeDataSource()
+        highestDifferentInApartmentRent =  GetHighestDifferentInApartmentRent(fakeDataSource)
+    }
+
+    @Test
+    fun should_ReturnNull_When_givenEmptyList(){
```

Comment:
I suggest to use this tests naming convention:
```suggestion
    fun `should return null when givenEmptyList`(){
```